src/services/loginMenu.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+
+import LoginAuth from '../services/firebase';
+
+export default function ResponsiveDialog() {
```

Comment:
1. Please move this file to `src/components/`
2. Rename the file to match the component name, e.g. `Login.js` or `LoginMenu.js`

Reply:
renamed to LoginMenu.js